src/syslog_mcp/tools/summary_generators.py

```
@@ -0,0 +1,375 @@
+        lines.append(f"**Last Activity:** {last_seen}")
+    lines.append("")
+    
+    # Activity analysis with timeline anomalies
+    lines.append("## 📈 ACTIVITY ANALYSIS")
+    
+    # Check for unusual activity patterns
+    if activity_timeline:
+        log_counts = [entry["log_count"] for entry in activity_timeline if entry["log_count"] > 0]
+        if log_counts:
+            avg_activity = sum(log_counts) / len(log_counts)
+            max_activity = max(log_counts)
```

Ruff (0.12.2)

Local variable `max_activity` is assigned to but never used

Remove assignment to unused variable `max_activity`

(F841)